### PR TITLE
Add Result.status Provisional

### PR DESCRIPTION
**Pull request by ottonomy**
The gap between "OnHold" with its somewhat negative connotation and "Completed" is big enough to squeeze in another possibility, "Provisional". This carries the expectation of an eventual completed result, but perhaps has not passed all administrative checks that will complete the status.

See existing terms: https://www.imsglobal.org/spec/ob/v3p0#org.1edtech.ob.v3p0.resultstatustype.class

I have some use cases for this for self-claimed badges that are not yet fully confirmed by their issuer, but may be useful to circulate in any case. Consumers SHOULD view this as an incomplete status, but they MAY consider it satisfactory.

### CODE REVIEW

**Comment by amiller-ims** [comment]
+1 awesome suggestion

**Comment by xaviaracil** [comment]
Ok to merge, but let's wait for WG approval in next meeting

**Comment by xaviaracil** [comment]
Merge approved by the working group, with a suggestion to add a chapter in the implementation guide about best practices using this enumeration.

**Review by xaviaracil — APPROVED** [review]

**Comment by xaviaracil** [comment]
Suggestion gathered in issue #562

**Pull request merged**
